index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Using Bootstrap</title>
+  </head>
+  <header>
+    <nav class="navbar fixed-top navbar-light bg-light">
+      <div class="logo">My Logo</div>
+      <div class="navbar-container">
+        <a class="navbar-brand" href="#">Portfolio</a>
+        <a class="navbar-brand" href="#">About</a>
+        <a class="navbar-brand" href="#">Contact</a>
+      </div>
+    </nav>
+  </header>
+
+  <section class="sectionA">
+    <div class="card card-1" style="width: 50rem">
+      <div class="card-body">
+        <h5 class="card-title">David Ranger</h5>
+        <h6 class="card-subtitle mb-2 text-muted">Software Developer</h6>
+        <p class="card-text">
+          I am a software developer. Please take some time to view through my
+          projects and experiences.
+        </p>
+        <a href="#" class="card-link">Card link</a>
+        <a href="#" class="card-link">Another link</a>
+      </div>
+    </div>
+  </section>
+
+  <section class="sectionB">
+    <div>
+      <div class="card card-2" style="width: 50rem">
+        <img src="./images/Portfolio1.png" class="card-img-top img" alt="..." />
+        <div class="card-body">
+          <p class="card-text">
+            Some quick example text to build on the card title and make up the
+            bulk of the card's content.
+          </p>
+        </div>
```

Comment:
You are doing a good job so far, May I just kindly recommend that it's always good to use a descriptive class name for your classes so that another developer who may be maintaining your code will not get lost and not prone to mistakes.